src/portal-components/DetailsPage.tsx

```diff
@@ -309,8 +330,56 @@ export default class DetailsPage extends React.Component<
     })
   }
 
-  renderSynapseConfigArray = () => {
+  renderTabs() {
+    const { tabIndex } = this.state
+    return(
+      <div className="tab-groups">
+        <span className={tabIndex === 0 ? "tab-item-active" : "tab-item"} onClick={(e) => {
+          this.setState({tabIndex: 0})
+        }}><StudySVG></StudySVG>Study Details</span>
```

Comment:
This is currently hard-coded to support a Study details page.  An enhancement would be to include the tabs definition as part of the config.

For example, instead of a hard-coded icon, this could use an icon type/value string and IconOptions that are used elsewhere (then this might become <Icon iconOptions={iconOptions} value={iconValue} type={type} /> for every tab config).

Reply:
And if icon fill color is presenting an additional complexity, let's not customize the color for now.